security/src/main/scala/pl/touk/nussknacker/ui/security/api/AuthenticationConfiguration.scala

```
@@ -28,7 +28,7 @@ object AuthenticationConfiguration {
   val rulesConfigurationPath = "rules"
 
   def getUsers(config: Config): List[ConfigUser] = config.as[List[ConfigUser]](usersConfigurationPath)
-
+  def getUsersOrEmptyList(config: Config): List[ConfigUser] = config.as[Option[List[ConfigUser]]](usersConfigurationPath).getOrElse(Nil)
```

Why not to have one methods instead?

---

Because for BasicAuth we require the users field on file - so we want to throw an exception when the field is missed.

---

Hmm, after thinking we can do one method -> check it right now.